great_expectations_cloud/logging_cfg.py

```diff
@@ -54,5 +54,27 @@ def configure_logger(log_level: LogLevel, log_cfg_file: pathlib.Path | None) ->
         logging.config.dictConfig(dict_config)
         LOGGER.info(f"Configured logging from file {log_cfg_file}")
     else:
-        logging.basicConfig(level=log_level.numeric_level)
-        LOGGER.info(f"Configured logging at level {log_level}")
+        logDirectory = pathlib.Path("logs")
+        if not logDirectory.exists():
+            pathlib.Path(logDirectory).mkdir()
+
+        logger = logging.getLogger()
+        formatter = logging.Formatter(
+            "%(asctime)s | %(name)s | %(lineno)d | %(levelname)s: %(message)s"
+        )
+        logger.setLevel(logging.DEBUG)
+
+        # The StreamHandler writes logs to stderr based on the provided log level
```

Comment:
In order to write to both stderr and the file, I added 2 logging handlers--one that writes to stderr and shows up on the terminal and one that writes to the files. 

Maybe eventually we can simplify to having just the files, but I think it's helpful to have a user-configurable logging level for output to stderr.